lib/basedriver/driver.js

```diff
@@ -140,6 +171,14 @@ class BaseDriver extends MobileJsonWireProtocol {
   // and ensuring that we execute commands one at a time. This method is called
   // by MJSONWP's express router.
   async executeCommand (cmd, ...args) {
+    // get start time for this command, and log in special cases
+    let startTime = Date.now();
```

Comment:
Yeah, probably not necessary. We could, though, just record the elapsed time unless we care about when they started and stopped for some reason.